be-a-managing-director/readme.md

```diff
@@ -0,0 +1,66 @@
+# How to be a Managing Director
+
+Great, so you’re the Managing Director of New Atoms, congratulations🎉!
+
+Lets take apart what that means for a second, in reverse order:
+
+1. Managing: Having executive control. Meaning you are the person that needs to deal with shit.
+2. Director: This means the person who is in charge. Meaning you are the person responsible for everything going well.
```

Comment:
Im missing a part of giving direction? something like this Director: This means the person who is in charge. Meaning you are the person responsible for deciding where to go and to make sure all the shit that is not in the capabilities of the employees happens and goes well.